hepcrawl/parsers/elsevier.py

```diff
@@ -0,0 +1,615 @@
+            "./{}/item-info/copyright[@type]/text()".format(self.top_level_document_type)
+        ).extract_first()
+        if not copyright_holder:
+            copyright_type = self.root.xpath(
+                "./{}/item-info/copyright/@type".format(self.top_level_document_type)
+            ).extract_first()
+            copyright_holder = COPYRIGHT_MAPPING.get(copyright_type)
+
+        return copyright_holder
+
+    @property
+    def copyright_statement(self):
+        copyright_statement = self.root.xpath("./RDF/Description/copyright/text()").extract_first()
+        if not copyright_statement:
+            copyright_statement = self.root.xpath(
+                "./{}/item-info/copyright[@type]/text()".format(self.top_level_document_type)
+            ).extract_first()
+
+        return copyright_statement
+
+    @property
+    def copyright_year(self):
+        copyright_year = self.root.xpath(
+            "./{}/item-info/copyright[@type]/@year".format(self.top_level_document_type)
+        ).extract_first()
+
+        return maybe_int(copyright_year)
+
+    @property
+    def dois(self):
+        doi = self.root.xpath(
+            "./{}/item-info/doi/text()".format(self.top_level_document_type)
+        ).extract_first()
+        return [{'doi': doi, 'material': self.material}]
+
+    @property
+    def document_type(self):
+        if self.is_conference_paper:
+            document_type = 'conference paper'
+        else:
+            document_type = 'article'
+
+        return document_type
+
+    @property
+    def is_conference_paper(self):
+        """Decide whether the article is a conference paper."""
+        if self.article_type == "conference paper":
+            return True
+        journal_issue = self.root.xpath("./RDF/Description/issueName/text()").extract_first()
+        if journal_issue:
+            is_conference = re.findall(r'proceedings|proc.', journal_issue.lower())
+            return bool(is_conference)
+        return False
+
+    @property
+    def journal_title(self):
+        jid = self.root.xpath(
+            './{}/item-info/jid/text()'.format(self.top_level_document_type)
+        ).extract_first(default='')
+        publication = self.root.xpath(
+            './RDF/Description/publicationName/text()'
+        ).extract_first(default=jid)
+        publication = re.sub(" [S|s]ection", '', publication).replace(',', '').strip()
+        return publication
+
+    @property
+    def journal_issue(self):
+        journal_issue = self.root.xpath("./serial-issue/issue-info/issue-first/text()").extract_first()
+
+        return journal_issue
+
+    @property
+    def journal_volume(self):
+        journal_volume = self.root.xpath("./RDF/Description/volume/text()").extract_first()
+
+        return journal_volume
+
+    @property
+    def keywords(self):
+        keywords = self.root.xpath(
+            "./{}/head/keywords[not(@abr)]/keyword/text/text()".format(self.top_level_document_type)
```

Comment:
why the `[not(@abr)]`. What do those represent?

Reply:
These are abbreviations used in article, I don't think that they should be included